scripts/prepare.py

```diff
@@ -0,0 +1,80 @@
+import argparse
+import datetime
+import logging
+from glob import glob
+from pathlib import Path
+from shutil import copy
+
+
+def main():
+    parser = argparse.ArgumentParser(
+        description='Prepares package files for building'
+    )
+    parser.add_argument(
+        'sources_dir',
+        help='Directory which contains source files for your library',
+        type=Path
+    )
+    parser.add_argument(
+        'desc_file', nargs='?',
+        metavar='DESCRIPTION_file',
+        help='''The path to the DESCRIPTION file for your package,
+            ./DESCRIPTION by default. The file may use placeholder
+            tags ##VERSION## and ##DATE## which will be replaced with
+            current information. Note that if the ##VERSION## tag is
+            present, the version argument is required''',
+        default='./DESCRIPTION',
+        type=Path
+    )
+    parser.add_argument(
+        '-v', '--version', metavar='version',
+        help='The version of your package',
+        default=''
+    )
+    parser.add_argument(
+        '-o', '--out-dir', dest='out_dir', metavar='output_dir',
+        help='The directory to paste the files to, ./out/ by default',
```

Comment:
```suggestion
        help='The directory for generated files, ./out/ by default',
```

Reply:
This specifically references original library files

Reply:
But for me as the user it is not clear what `paste` means, to paste what ?